python/hail/dataset.py

```diff
@@ -4692,13 +4689,13 @@ def skat(self, variant_keys, single_key, weight_expr, y, covariates=[], logistic
 
```

Comment:
I can't comment above because it's outside of the changes zone, but there's a bit that says:
>          For each variant, missing genotypes are imputed as the mean of all non-missing genotypes.

Which should now say "non-missing input values", or something like that, right?

Reply:
Good catch.  I fixed this in a number of places.